Dockerfile-73

```diff
@@ -132,7 +132,7 @@ RUN cp /etc/php/7.0/mods-available/* $CONF_PHPMODS && \
     # Override default ini values for both CLI + FPM \
     phpenmod overrides && \
     # Enable NewRelic via Ubuntu symlinks, but disable in file. Cross-variant startup script uncomments with env vars.
```

Comment:
Does this comment need to be updated?

Reply:
It's the correct comment, because of the way newrelic is enabled or disabled

Reply:
🆗